content/en/security/application_security/threats/add-user-info.md

```diff
@@ -714,29 +714,58 @@ Once saved, the rule is deployed to instances of the service that have Remote Co
+<div class="alert alert-info">All modes only affect automated instrumentation. The modes don't apply to manual collection. Manual collection is confiigured using an SDK, and those settings are not overridden by automated instrumentation.</div>
 
-To configure automatic user event tracking mode, you can set the environment variable <code>DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING</code> to <code>safe</code> or <code>extended</code>. By default, the tracer library uses the <code>safe</code> mode.
+Datadog libraries allow you to configure auto-instrumentation by using the `DD_APPSEC_AUTO_USER_INSTRUMENTATION_MODE` environment variable with the short name for the mode: `ident`|`anon`|`disabled`.
 
-**Note**: There could be cases in which the trace library won't be able to extract any information from the user event. The event would be reported with empty metadata. In those cases, we recommend using the [SDK](#adding-business-logic-information-login-success-login-failure-any-business-logic-to-traces) to manually instrument the user events.
+The default mode is `identification` mode (short name: `ident`).
+
+For example, `DD_APPSEC_AUTO_USER_INSTRUMENTATION_MODE=anon`.
+
+### Deprecated modes
+
+<div class="alert alert-info">Previous modes are deprecated, but compatibility will be maintained until the next major release.</div>
+
+The following modes are deprecated:
+
+- `safe` mode: The trace library does not include any PII information on the events metadata. The tracer library tries to collect the user ID, and only if the user ID is a valid [GUID][10]
+- `extended` mode: The trace library tries to collect the user ID, and the user email. In this mode, Datadog does not check the type for the user ID to be a GUID. The trace library reports whatever value can be extracted from the event.
+
+**Note**: There could be cases in which the trace library won't be able to extract any information from the user event. The event would be reported with empty metadata. In those cases, use the [SDK](#adding-business-logic-information-login-success-login-failure-any-business-logic-to-traces) to manually instrument the user events.
 
 ## Disabling automatic user activity event tracking
 
-If you wish to disable the detection of these events, you should set the environment variable <code>DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING</code> to <code>disabled</code>. This should be set on the application hosting the Datadog Tracing Library, and not on the Datadog Agent.
+If you wish to disable the detection of these events, you should set the environment variable `DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING_ENABLED` to `false`. This should be set on the application hosting the Datadog Tracing Library, and not on the Datadog Agent.
+
+The previous environment variable was named `DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING`.
```

Comment:
Perhaps it might make sense to reiterate somewhere that this variable was replaced by `DD_APPSEC_AUTO_USER_INSTRUMENTATION_MODE` and compatibility will be maintained (as mentioned above).